components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -92,6 +98,32 @@ enum class SolanaTokenInstruction {
   kInitializeMint2
 };
 
+struct SolanaSignatureStatus {
+  SolanaSignatureStatus() = default;
+  SolanaSignatureStatus(uint64_t slot,
+                        uint64_t confirmations,
+                        const std::string& err,
+                        const std::string& confirmation_status);
+  ~SolanaSignatureStatus() = default;
+  SolanaSignatureStatus(const SolanaSignatureStatus&) = default;
+  bool operator==(const SolanaSignatureStatus&) const;
+  bool operator!=(const SolanaSignatureStatus&) const;
+
+  base::Value ToValue() const;
+  static absl::optional<SolanaSignatureStatus> FromValue(
+      const base::Value& value);
+
+  // The slot the transaction was processed.
+  uint64_t slot = 0;
+  // Number of blocks since signature confirmation.
+  uint64_t confirmations = 0;
```

Comment:
can we comment that choosing `uint64_t`  instead of `size_t` is because we need max container for server response?
ex. 64 bit system server and 32 bit system client

Reply:
Addressed in https://github.com/brave/brave-core/commit/da16927fc1d1fa35a5287009706993ad47224fde